src/puya/awst/function_traverser.py

```diff
@@ -1,5 +1,6 @@
 import puya.awst.visitors
 from puya.awst import nodes as awst_nodes
+from puya.awst.visitors import T
```

Comment:
Shouldn't need this?

Reply:
Just a remnant of pycharm auto creating the missing abstract member